documentation/modules/ref-python-example.adoc

```diff
@@ -37,3 +37,28 @@ class HelloWorld(MessagingHandler):
 Container(HelloWorld("amqps://_messaging-route-hostname_:443", "myqueue")).run()
 ----
 
+== Known issue with creating a subscriber on a hierarchical topic
+
+A known issue exists where creating a subscriber on a hierarchical topic in {ProductName} causes the broker to instead create it as a competing consumer (handling the address like a queue rather than a topic).
+
+The workaround for this issue involves setting the capability `"topic"` in the source.
+
+.Procedure
+
+. In the `simple_recv.py` file, add the following lines:
+
+[source,python,options="nowrap",subs="+quotes,attributes"]
+----
+class CapabilityOptions(ReceiverOption):
+    def apply(self, receiver):
+        receiver.source.capabilities.put_object(symbol("topic"))
+----
+
+. Modify the following line to add `options=CapabilityOptions()`:
+
+[source,python,options="nowrap",subs="+quotes,attributes"]
+----
+def on_start(self, event):
+        event.container.create_receiver(self.url, options=CapabilityOptions())
```

Comment:
There are 2 similar python examples in our documentation:
- common/client-example1.py (same as: contributing/common/client-example1.py, service_admin/common/client-example1.py, tenant/common/client-example1.py, address_space_admin/common/client-example1.py, messaging_app_developer/common/client-example1.py
- modules/ref-python-example.adoc

For this file (modules/ref-python-example.adoc) the line needs to be: 
`event.container.create_receiver(conn, self.address, options=CapabilityOptions())`
For all other locations `event.container.create_receiver(self.url, options=CapabilityOptions())`

Reply:
Fixed